app/referee/tests/test_job_submitter.py

```diff
@@ -0,0 +1,111 @@
+import os
+import time
+from unittest.mock import Mock, patch
+
+import dask.distributed as dd
+import pytest
+from django.contrib.contenttypes.models import ContentType
+from django.core.management import call_command
+from django.db import transaction
+
+from core import models
+from referee import job_submitter, tasks
+
+
+def test_start_cluster():
+    config_file = f"{os.path.dirname(os.path.abspath(__file__))}/jobqueue_test.yaml"
```

Comment:
```suggestion
    config_file = os.path.join(os.path.dirname(os.path.abspath(__file__)), "jobqueue_test.yaml")
```
also check out pathlib